src/coreclr/utilcode/util.cpp

```diff
@@ -237,7 +237,7 @@ namespace
         StackSString ssDllName;
         if ((wszDllPath == nullptr) || (wszDllPath[0] == W('\0')) || fIsDllPathPrefix)
         {
-#ifndef TARGET_UNIX
+#ifdef FEATURE_COMINTEROP
```

Comment:
Nit: This is unrelated to COM interop. This path is used to load profilers or unmanaged symbol readers. It happens to use COM, but not COM interop. If we were to build CoreCLR runtime without COM interop, we would still want this path to work.

Reply:
It may be nice to change `#ifndef TARGET_UNIX` to `#ifdef HOST_WINDOWS` here.

Reply:
Ah, I had switched this because `FindInprocServer32UsingCLSID` in `utilcode.h` is under `FEATURE_COMINTEROP`. I'll make them `HOST_WINDOWS`.